trade/trader_stakecube.js

```diff
@@ -522,19 +538,21 @@ module.exports = (
       const paramString = `pair: ${pair}`;
       const pair_ = formatPairName(pair);
 
-      let book;
+      let scData;
       try {
-        book = await stakeCubeApiClient.orderBook(pair_.pair);
+        scData = await stakeCubeApiClient.orderBook(pair_.pair);
 
-        if (book.errorMessage) {
-          log.warn(`API request getOrderBook(${paramString}) of ${utils.getModuleName(module.id)} module failed. ${book.errorMessage}`);
+        if (scData.error) {
```

Comment:
It's not necessary to check this as in case of error, function will be try-catched.
In case of error, we already logged it in `stakecube_api`.